ScoutSuite/providers/gcp/resources/iam/keys.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+from ScoutSuite.providers.base.configs.resources import Resources
+
+class Keys(Resources):
+    def __init__(self, iam_facade, project_id, service_account_email):
+        self.iam_facade = iam_facade
+        self.project_id = project_id
+        self.service_account_email = service_account_email 
+
+    async def fetch_all(self):
+        raw_keys = await self.iam_facade.get_keys(self.project_id, self.service_account_email)
+        for raw_key in raw_keys.get('keys', []):
+            key_id, key = self._parse_key(raw_key)
+            self[key_id] = key
+
+    def _parse_key(self, raw_key):
```

Comment:
[Same](https://github.com/nccgroup/ScoutSuite/pull/229#discussion_r263382611), would rename to `_parse`.